src/components/cyclejs/cyclejs.js

```diff
@@ -0,0 +1,37 @@
+import { img } from '@cycle/dom'
+import xs from 'xstream'
+import delay from 'xstream/extra/delay'
+import { WIRE_TIMEOUT } from '../../config'
+
+const stopEvent = { stop: true }
+
+const addDelay = stream$ => stream$ && stream$.compose(delay(WIRE_TIMEOUT))
+
+export default ({ MUSIC$, NOTE$, HTTP$ }) => {
+  const className = '.cyclejs'
+
+  const start$ = xs.merge(
```

Comment:
I see that CycleJS is a Wire with a different DOM representation.
In a real life project we would merge some code.
But I bet this is better to let it be that way for the conference, this is easier to understand.